src/bgfx_p.h

```diff
@@ -206,6 +206,7 @@ namespace stl = std;
 #	include <windows.h>
 #endif // BX_PLATFORM_*
 
+#define BGFX_MAX_FRAME_LATENCY 3
```

Comment:
Move it into config.h, and rename it to `BGFX_CONFIG_MAX_FRAME_LATENCY`. Also add `#ifdefs` around it so it can be specified via build system.

Reply:
Done 👍